topologies/datacenter-latest/labguides/source/bgp-with-cvp.rst

```diff
@@ -0,0 +1,170 @@
+Layer 3 Leaf-Spine
+==================
+
+.. image:: images/cvp-l3ls/l3ls_1.png
+   :align: center
+   :width: 50 %
+
+.. note:: Did you know the “bgp” script is composed of Python code that
```

Comment:
@jrecchia1029 Can probably remove this note as the user isn't leveraging the login.py code to configure the lab.